rootfs/api/models/app.py

```diff
@@ -657,7 +681,7 @@ def verify_application_health(self, **kwargs):
             time.sleep(1)
 
         # Endpoint did not report healthy in time
-        if ('response' in locals() and response.status_code == 404) or failed:
+        if (('response' in locals() and response.status_code == 404) or failed) and not is_experimental_native_ingress():
```

Comment:
> make the documentation crystal clear that if you want to BYO ingress controller, you need to have that installed before you can rightfully expect Workflow to start working.

I think that's a reasonable approach for an experimental feature. We can change it if this is tripping users up.